eth/handler.go

```diff
@@ -99,10 +100,11 @@ type handler struct {
 	checkpointNumber uint64      // Block number for the sync progress validator to cross reference
 	checkpointHash   common.Hash // Block hash for the sync progress validator to cross reference
 
-	database ethdb.Database
-	txpool   txPool
-	chain    *core.BlockChain
-	maxPeers int
+	database        ethdb.Database
+	txpool          txPool
+	chain           *core.BlockChain
+	maxPeers        int
+	disableTxGossip bool
```

Comment:
Adding a blank line above this is an easy way to not reformat the other unrelated parts.

Reply:
Done.